scripts/jenkins/ed.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Additionally we probably want to validate the input at least a bit, to protect against the Makefile accidentally writing somewhere crazy here and to make sure the source actually exists.